release_notes/ocp-4-7-release-notes.adoc

```diff
@@ -793,16 +844,43 @@ This dashboard provides API server metrics, such as:
 
 // TODO: Link to troubleshooting using this dashboard section once it is available
 
-[id="ocp-4-7-monitoring-etcd-alerts"]
-==== New etcd alerts
+[id="ocp-4-7-monitoring-namespace-by-pod-and-pod-total-Grafana-dashboards"]
+==== Enable Namespace (Pods) and Pod Kubernetes networking dashboards in Grafana
+
+The *Namespace (Pods)* and *Pod* Kubernetes networking dashboards are now enabled in Grafana. You can access the *Namespace (Pods)* and *Pod* dashboards from the web console by navigating to *Monitoring* -> *Dashboards* -> *Grafana UI*.
+
+These dashboards provide networking metrics, such as:
+
+* Current rate of bytes received per namespace or per pods
```

Comment:
You could potentially change "Current rate" to "The current rate...". The same goes for the other items starting with "Rate..." below, each sentence could start with "The".

Reply:
You could change "per pods" to "per pod" in these two lines.

Reply:
> You could potentially change "Current rate" to "The current rate...". The same goes for the other items starting with "Rate..." below, each sentence could start with "The".

Going to keep without the "The" to match the "API Performance" release note above.